src/components/premier-league/RegistrationForm.tsx

```
@@ -0,0 +1,1277 @@
+                      <div
+                        key={option.id}
+                        className={`relative cursor-pointer minimal-radius border-2 overflow-hidden transition-all duration-500 shadow-2xl hover-lift hover-scale ${
+                          formData.league_type === option.id
+                            ? `${option.colors.border} ${option.colors.bg} shadow-lg ring-4 ${option.colors.badgeRing}`
+                            : `border-gray-600/50 ${option.colors.hover} hover:shadow-xl`
+                        }`}
+                        onClick={() =>
+                          handleInputChange("league_type", option.id)
+                        }
+                      >
+                        <input
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)